### PR TITLE
Ap user profile

### PR DESCRIPTION
**Pull request by APatel-AI**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                </div>
+              </div>
+            </li>
+          <% end %>
+
+        </ul>
+        <div class="card-body">
+
+          <div class="card-body">
+            <%= render "comments/form", comment: photo.comments.build %>
```
Comment (dylansarikas) [inline app/views/users/show.html.erb:35]: Very concise!

**Review by dylansarikas — APPROVED** [review]
Looks good!!

File: app/views/comments/_form.html.erb [hunk 2]
```diff
@@ -11,22 +11,13 @@
...
-  <div>
-    <%= form.label :author_id, style: "display: block" %>
-    <%= form.text_field :author_id %>
-  </div>
-
-  <div>
-    <%= form.label :photo_id, style: "display: block" %>
-    <%= form.text_field :photo_id %>
-  </div>
+  <%= form.hidden_field :photo_id %>
```
Comment (JoseMDO) [inline app/views/comments/_form.html.erb:14]: Great job using hidden inputs!

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
```
Comment (JoseMDO) [inline app/views/users/show.html.erb:21]: If we dont need an image tag, its always good to either delete it or add default values so it does not mess with the accessibility of the app :)

File: config/routes.rb [hunk 4]
```diff
@@ -1,10 +1,16 @@
...
+
+
   devise_for :users
   resources :comments
   resources :follow_requests
   resources :likes
   resources :photos
 
+  # resources :users, only: :show
+  get "/:username" => "users#show", as: :user
```
Comment (JoseMDO) [inline config/routes.rb:14]: Nice custom route!

**Review by JoseMDO — APPROVED** [review]
Great job Arpan!

**Pull request merged**